snc.sh

```diff
@@ -74,6 +74,20 @@ function run_preflight_checks() {
         if ! virsh -c ${LIBVIRT_URI} uri >/dev/null; then
                 preflight_failure  "libvirtd is not listening for plain-text TCP connections, see https://github.com/openshift/installer/tree/master/docs/dev/libvirt#configure-libvirt-to-accept-tcp-connections"
         fi
+
+	#Check if default libvirt network is present
+	if virsh -c ${LIBVIRT_URI} net-info default &> /dev/null; then
+		echo "default network is available"
+	
+		#Check if default libvirt network is Active
+		if [[ $(virsh -c qemu:///system net-info default | awk '{print $2}' | sed '3q;d') == "no" ]]; then
+			echo "Default network is not active. Exiting now!"
+			exit 1
+		fi
+	else
+		echo "Default libvirt network is not available. Exiting now!"
+		exit 1
+	fi
```

Comment:
Would be simpler as 
```
if ! virsh -c ${LIBVIRT_URI} net-info default &> /dev/null; then
	echo "Default libvirt network is not available. Exiting now!"
	exit 1
fi
echo "default network is available"

#Check if default libvirt network is Active
if [[ $(virsh -c qemu:///system net-info default | awk '{print $2}' | sed '3q;d') == "no" ]]; then
	echo "Default network is not active. Exiting now!"
	exit 1
fi
```
This saves some indentation levels, and the nominal (no error) case is easier to follow.
For the "active" check, I'd just use something like `virsh -c qemu:///system net-info default |grep '^Active: *yes`, this is easier to read, and feels more robust.